src/components/lists/ShotList.vue

```diff
@@ -401,6 +416,27 @@ export default {
       )
     },
 
+    visibleColumns () {
```

Comment:
Not sure if it's the best way to count all the columns in a table but that's all I could think of. I'm open to suggestions to enhance this part if there is any.